Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.8
+
+RUN useradd -m appuser
+
+WORKDIR /home/appuser 
+USER appuser 
+COPY ./ /home/appuser
```

Comment:
We want to copy the `poetry.lock` and `pyproject.toml` files first, install the deps, and then copy everything else. This way we don't invalidate the install cache by changing files

Reply:
This has some additional opinions https://stackoverflow.com/questions/53835198/integrating-python-poetry-with-docker